src/sentry/api/endpoints/oauth_userinfo.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework.authentication import get_authorization_header
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, control_silo_endpoint
+from sentry.models import ApiToken, UserEmail
+
+
+@control_silo_endpoint
+class OAuthUserInfoEndpoint(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    def get(self, request: Request) -> Response:
+        try:
+            access_token = get_authorization_header(request).split()[1].decode("utf-8")
```

Comment:
is it always the case that the access token is the 2nd element from `get_authorization_header().split()`?

Reply:
As far as I can tell yes, using the code for `get_authorization_header()` as a reference ([source here](https://github.com/getsentry/sentry/blob/2ee3b7f248bbb196041bf2cd3b8cf7754feeea05/)), if the length of the split headers is not equal to 2 then we raise an error, and if it is 2, we authenticate with the second element in the array.

Reply:
maybe this should be more explicit then. i see you're catching the `IndexError` but it's not obvious that the access token HAS to have two parts when split